mina-daemon/templates/deployment.yaml

```diff
@@ -31,6 +31,20 @@ spec:
       serviceAccountName: {{ .Release.Name }}
       initContainers:
         {{ if .Values.node.libp2pKeys.enabled -}}
+        {{ if .Values.node.libp2pKeys.create -}}
```

Comment:
here too, `{{ if and ... }}` sounds more appropriate

Reply:
same here
the logic is the following
```
if libp2pKeys.enabled
   if libp2pKeys.create
       do xxxxxx # Case where it is autogenerated
   else
       do xxxxxx # Case where we import libp2pKeys from secret
   fi
fi
```